app/code/Magento/SalesGraphQl/Model/Resolver/CreditMemoComments.php

```diff
@@ -0,0 +1,55 @@
+<?php
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+declare(strict_types=1);
+
+namespace Magento\SalesGraphQl\Model\Resolver;
```

Comment:
To organize file structure can we make this namespace like Magento\SalesGraphQl\Model\Resolver\CreditMemo

Reply:
Similar for other sub resolvers for credit memo fields